README.md

```diff
@@ -264,7 +264,9 @@ that caused Neoformat to be invoked.
   - [`gleam format`](https://github.com/gleam-lang/gleam/)
 - Go
   - [`gofmt`](https://golang.org/cmd/gofmt/),
-    [`goimports`](https://godoc.org/golang.org/x/tools/cmd/goimports)
+    [`goimports`](https://godoc.org/golang.org/x/tools/cmd/goimports),
```

Comment:
could you also update the in vim docs, `docs/neoformat.txt`?

otherwise looks good

Reply:
Updated